operator/src/test/python/test_wdt_mii_filter.py

```diff
@@ -364,6 +364,7 @@ def test_customizeIstioReplicationChannel_version_1_10(self):
       istiorepl_nap = model['topology']['ServerTemplate']['cluster-1-template']['NetworkAccessPoint']['istiorepl']
       self.assertIsNotNone(istiorepl_nap, "Expected a NAP named \'istiorepl\' to be defined")
 
+      self.assertEqual(istiorepl_nap['TunnelingEnabled'], 'false')
```

Comment:
This file seems to be missing the copyright header altogether...

Reply:
done